pycqed/analysis/measurement_analysis.py

```diff
@@ -6720,9 +6720,10 @@ def fit_all(self, plot=False, return_stds=False, sign_of_peaks=None):
         if plot:
             fig, ax = plt.subplots()
             if return_stds:
-                ax.errorbar(delays/1e-9, fitted_freqs/1e6, yerr=fitted_stds/1e6)
+                ax.errorbar(delays/1e-9, self.fitted_freqs/1e6,
+                            yerr=fitted_stds/1e6)
             else:
-                ax.plot(delays/1e-9, fitted_freqs/1e6)
+                ax.plot(delays/1e-9, self.fitted_freqs/1e6)
```

Comment:
This looks like some merge bug. Here the attribute self.fitted_freqs is used but it is only assigned at lines 6731 below. Lines 6731-32 probably need to be moved up before "if plot:"